### PR TITLE
[MST-678] Roll out exam resume and grouped attempt features

### PR DESCRIPTION
**Pull request by bseverino**
**Description:**

Remove references to `enable-exam-resume-proctoring-improvements`, and fully roll out the exam resume and grouped attempt features on the instructor dashboard.

**JIRA:**

[MST-678](https://openedx.atlassian.net/browse/MST-678)

**Pre-Merge Checklist:**

- [x] Updated the version number in `edx_proctoring/__init__.py` and `package.json` if these changes are to be released.
- [x] Described your changes in `CHANGELOG.rst`
- [ ] Confirmed Github reports all automated tests/checks are passing.
- [ ] Approved by at least one additional reviewer.

**Post-Merge:**

- [ ] Create a tag matching the new version number.

### CODE REVIEW

File: edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js [hunk 1]
```diff
@@ -45,21 +45,12 @@ edx = edx || {};
...
             this.model = new edx.instructor_dashboard.proctoring.ProctoredExamAttemptModel();
 
-            if (!this.enable_exam_resume_proctoring_improvements) {
-                this.collection = new edx.instructor_dashboard.proctoring.ProctoredExamAttemptCollection();
-                this.template_url = '/static/proctoring/templates/student-proctored-exam-attempts.underscore';
-            } else {
-                this.collection = new edx.instructor_dashboard.proctoring.ProctoredExamAttemptGroupedCollection();
-                this.template_url = '/static/proctoring/templates/student-proctored-exam-attempts-grouped.underscore';
-            }
+            this.collection = new edx.instructor_dashboard.proctoring.ProctoredExamAttemptCollection();
```
Comment (schenedx) [inline edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js:52]: I think you rolled out the wrong logic on javascript side. The `ProctoredExamAttemptGroupedCollection` is the one you want to keep. See https://github.com/edx/edx-proctoring/pull/785/files

**Review by schenedx — CHANGES_REQUESTED** [review]
See inline comment. I believe you retired the wrong code.

File: edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js (hunk 1 above)
Comment (bseverino) [reply edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js:52]: I realized this is pretty confusing in the diff, I probably should have noted - I removed the old `ProctoredExamAttemptCollection`, and renamed `ProctoredExamAttemptGroupedCollection`, which now points to `StudentProctoredExamAttemptsByCourse` with the [updated grouped attempt code](https://github.com/edx/edx-proctoring/pull/820/files#diff-1674e60a26bd85080524664f75371c1b08489500da4f2a8d3fa16f8a8e2598abL990).

**Review by schenedx — APPROVED** [review]
👍 This is also easier to review.

**Pull request merged**